test/index.test.js

```diff
@@ -130,7 +130,7 @@ describe('Given invalid input for ', () => {
 });
 
 describe('Catching errors in updateVpcConfig ', () => {
-  it('AWS api call describeVpcs fails', () => {
+  it('AWS api call describeVpcs fails', (done) => {
```

Comment:
It should be downloading the latest, it seems to get held up because of a timeout error.